projects/vtk.cmake

```diff
@@ -51,8 +51,8 @@ superbuild_add_project(vtk
     -DVTK_MODULE_ENABLE_VTK_TestingCore=YES
     -DVTK_OPENGL_HAS_EGL=${egl_enabled}
     -DVTK_SMP_ENABLE_SEQUENTIAL=OFF
-    -DVTK_SMP_ENABLE_STDTHREAD=OFF
-    -DVTK_SMP_ENABLE_TBB=ON
-    -DVTK_SMP_IMPLEMENTATION_TYPE=TBB
```

Comment:
TBB cannot be built because there's no way to make it a static lib.  
In the end, I think I'll have to support for shared libs in the python wheels anyway, but it wouldn't hurt to make it optional

Reply:
https://github.com/f3d-app/f3d-superbuild/pull/85